src/ivoc/symdir.cpp

```diff
@@ -502,14 +502,17 @@ void SymDirectoryImpl::load_section() {
     nrn_popsec();
 }
 
-void SymDirectoryImpl::load_mechanism(Prop* p, int type, const char* xarg) {
-    NrnProperty np(p);
-    if (np.is_point()) {
+void SymDirectoryImpl::load_mechanism(Prop* p, int vartype, const char* xarg) {
```

Comment:
This function is almost fully covered (and all changes of this PR are covered) via manual execution of test_mechstd6() in test/hoctests/tests/test_mechstd.py by navigating to ```sdata``` and ```SData[0]```.  A unit_test could be added to execute the function but it would require some small hoc setup and either a couple of mod files or else larger hoc fragments. I would need some help in developing such a unit_test.